website/blog/2022-09-06/index.md

```diff
@@ -165,6 +170,6 @@ At the moment, the HTTP requires a function-specific API key to execute it as se
 
```

Comment:
We'll want to right-click publish from Visual Studio. And I'm going back and forth on whether we just do the whole resource group / function app / function creation process from VS. And skip the portal altogether (???).

If you really wanted to get fancy, you could create an Azure Dev CLI script that would provision the whole thing out and just tell them "run this" first and then follow along with the article.